src/basic/xattr-util.c

```diff
@@ -103,17 +103,19 @@ int fgetxattr_malloc(
         }
 }
 
-int fgetxattrat_fake(
+/* Note: ret_fn should already be allocated for the usual xsprintf and /proc/self/fd/%i pattern. */
+static int fgetxattrat_fake_internal(
```

Comment:
this doesn't actually do any getxattr-like stuff, it just prepares things for it, hence maybe makt that clear in the name? `getxattrat_prepare()`?